src/Elements/utilities.jl

```diff
@@ -0,0 +1,35 @@
+["Helper functions."]
+
+"""
+Find the final configuration fo node `n`.
```

Comment:
For code syntax in markdown can we consistently use double backticks. It's much more noticeable when reading and I've been trying to change them in the codebases when updating surrounding code.